src/main/java/core/basesyntax/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private final Map<FruitTransaction.Operation, OperationHandler> operationHandlers;
+
+    public OperationStrategyImpl(Map<FruitTransaction.Operation,
+            OperationHandler> operationHandlers) {
+        this.operationHandlers = operationHandlers;
+    }
+
+    public void applyOperation(FruitTransaction transaction) {
```

Comment:
Strategy class should  return Handler, call method handle in the TransactionProcessorServiceImpl